internal/cmd/asyncapi/command_asyncapi_test_utils.go

```diff
@@ -0,0 +1,222 @@
+			},
+			CreateTagDefsFunc: func(ctx context.Context, localVarOptionals *srsdk.CreateTagDefsOpts) ([]srsdk.TagDefResponse, *http.Response, error) {
+				return []srsdk.TagDefResponse{
+					{Name: "Tag1"}, {Name: "Tag2"},
+				}, nil, nil
+			},
+			CreateTagsFunc: func(ctx context.Context, localVarOptionals *srsdk.CreateTagsOpts) ([]srsdk.TagResponse, *http.Response, error) {
+				return []srsdk.TagResponse{{TypeName: "Tag1"}, {TypeName: "Tag2"}}, nil, nil
+			},
+			GetByUniqueAttributesFunc: func(_ context.Context, typeName string, qualifiedName string, localVarOptionals *srsdk.GetByUniqueAttributesOpts) (srsdk.AtlasEntityWithExtInfo, *http.Response, error) {
+				if typeName == "kafka_topic" {
+					return srsdk.AtlasEntityWithExtInfo{Entity: srsdk.AtlasEntity{Attributes: map[string]any{"description": "kafka topic description"}}}, nil, nil
+				}
+				return srsdk.AtlasEntityWithExtInfo{}, nil, nil
+			},
+			ListFunc: func(_ context.Context, _ *srsdk.ListOpts) ([]string, *http.Response, error) {
+				return []string{"subject 1", "subject 2"}, nil, nil
+			},
+			ListVersionsFunc: func(_ context.Context, _ string, _ *srsdk.ListVersionsOpts) ([]int32, *http.Response, error) {
+				return []int32{1234, 4567}, nil, nil
+			},
+			GetSchemaByVersionFunc: func(_ context.Context, subject string, _ string, _ *srsdk.GetSchemaByVersionOpts) (srsdk.Schema, *http.Response, error) {
+				if subject == "subject2" {
+					return srsdk.Schema{
+						Subject:    "subject1",
+						Version:    1,
+						Id:         1,
+						SchemaType: "PROTOBUF",
+						Schema:     `syntax = "proto3"; package com.mycorp.mynamespace; message SampleRecord { int32 my_field1 = 1; double my_field2 = 2; string my_field3 = 3;}`,
+					}, nil, nil
+				}
+				return srsdk.Schema{
+					Subject:    "subject1",
+					Version:    1,
+					Id:         1,
+					SchemaType: "avro",
+					Schema:     `{"doc":"Sample schema to help you get started.","fields":[{"doc":"The int type is a 32-bit signed integer.","name":"my_field1","type":"int"},{"doc":"The double type is a double precision(64-bit) IEEE754 floating-point number.","name":"my_field2","type":"double"},{"doc":"The string is a unicode character sequence.","name":"my_field3","type":"string"}],"name":"sampleRecord","namespace":"com.mycorp.mynamespace","type":"record"}`,
+				}, nil, nil
+			},
+			GetSubjectLevelConfigFunc: func(_ context.Context, _ string, _ *srsdk.GetSubjectLevelConfigOpts) (srsdk.Config, *http.Response, error) {
+				return srsdk.Config{CompatibilityLevel: BackwardCompatibilityLevel}, nil, nil
+			},
+			GetTopLevelConfigFunc: func(ctx context.Context) (srsdk.Config, *http.Response, error) {
+				return srsdk.Config{CompatibilityLevel: BackwardCompatibilityLevel}, nil, nil
+			},
+			GetTagsFunc: func(_ context.Context, _, _ string) ([]srsdk.TagResponse, *http.Response, error) {
+				return []srsdk.TagResponse{
+					{
+						TypeName: "Public",
+					},
+				}, nil, nil
```

Comment:
Rule of thumb: If a struct has one element in it, it's a bit easier to read if it's collapsed.
```suggestion
				return []srsdk.TagResponse{{TypeName: "Public"}}, nil, nil
```